src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +135,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
 
         ObservableList<String> usersList = FXCollections.observableArrayList(
```

Comment:
Might want to add a comment explaining why you're using fake data. This will become a moot point as you hook this up to the DB though.